test/sandbox.jl

```diff
@@ -0,0 +1,45 @@
+using AWSS3, Minio, FilePathsBase
```

Comment:
I don't really get the advantage of this as a separate script vs the minio setup in `runtests.jl`

Reply:
It's more about creating the bucket and a few names.  :shrug:   Up to you of course, but every time I needed to check something I needed this, figured I'd leave it around as a courtesy to the next person.  If you want it out it's out.

Reply:
hm, I'll leave it up to @mattBrzezinski or @omus. If we do keep it we should include it in the tests just to make sure it doesn't error/bitrot.

Reply:
Woah, woah, if you want to include it in tests than *I* definitely do not want to keep it :laughing: